cmd/k8s-operator/deploy/manifests/crds/tailscale.io_connectors.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I might do that in a separate PR.

Reply:
I will update this part once https://github.com/tailscale/tailscale/pull/10436 merges.

Reply:
In fact, since the Connector functionality is now opt-in, it should be ok to merge this PR as is and add the Helm chart CRD generation in a separate step.